tests/clients/test_evaluate_client.py

```diff
@@ -117,12 +117,17 @@ def test_metrics_reporter_evaluate() -> None:
     reporter = JsonReporter()
     evaluate_client = MockEvaluateClient(loss=test_loss, metrics=test_metrics, reporters=[reporter])
     evaluate_client.evaluate([], {})
+    print(reporter.metrics)
```

Comment:
We can toss this print statement I think?

Reply:
good catch